src/app/_components/Lobby/_subcomponents/SetUpCard/SetUpCard.tsx

```diff
@@ -0,0 +1,158 @@
+import React from "react";
+import {
+	Card,
+	CardContent,
+	Typography,
+	TextField,
+	Button,
+	Box, CardActions,
+} from "@mui/material";
+import { useGame } from "@/app/_contexts/Game.context";
+import { useRouter, useSearchParams } from "next/navigation"
+import {ILobbyUserProps, ISetUpProps} from "@/app/_components/Lobby/LobbyTypes";
+
+const SetUpCard: React.FC<ISetUpProps> = ({
+  readyStatus,
+  owner,
+}) => {
+
+	const { sendMessage, lobbyState, connectedPlayer, sendLobbyMessage, gameState } = useGame();
+	const opponentUser = lobbyState ? lobbyState.users.find((u: ILobbyUserProps) => u.id !== connectedPlayer) : null;
+	// Extract the player from the URL query params
+	const router = useRouter();
+
+	//------------------------Additional functions------------------------//
+	const handleStartGame = async () => {
+		sendMessage("startGame");
+		router.push("/GameBoard");
+	};
+
+	//------------------------STYLES------------------------//
+	const setUpCard = {
```

Comment:
Lets convert these to all be properties in a `styles` object so that they're easily collapsable.